functions/Export-DbaScript.ps1

```diff
@@ -168,6 +168,8 @@ function Export-DbaScript {
                 Write-Message -Level Warning -Message "Setting ScriptBatchTerminator to true and also having BatchSeperarator as an empty or null string may produce unintended results."
             }
         }
+
+        $eol = [System.Environment]::NewLine
```

Comment:
something must be missing here -- export-dbainstance exports user objects and that particular script causes an issue

![image](https://user-images.githubusercontent.com/8278033/128680896-fbb61335-7eff-4bec-a753-4ea8c8c45272.png)

Reply:
It's the line before a CREATE PROCEDURE:

![image](https://user-images.githubusercontent.com/66946165/128684876-48a56dac-f359-4974-b857-b524bcc103dd.png)

Reply:
I think the single LF is in the code. If I use SSMS to "Modify" the procedure I get the same message. So we would have to change that in the resultung $scriptInFull. Don't know if we really want to do that.

Reply:
So you think something like a `$scriptInFull.Replace("`r",$eof)` is too dangerous or unnecessary? I can't decide.

Reply:
Wouldn't that change "rn" into "rrn"?

Reply:
yah sorry i was riffing, whatever it is that would replace LF (isn't ``` `r`n ``` CLRF?) with $eof

Reply:
This would work:
```
$scriptInFull = $scriptInFull -replace "(`r)?`n", $eol
```

Should we run that if a special config value is set? Or in any case? 
And this would have to be added in multiple places inside of Export-DbaScript and would not effect those branches that write to a file via Scripter directly.

Reply:
@niphlod what do you think? I'm leaning not at all or any case but mostly any case.